README.md

```diff
@@ -134,7 +133,6 @@ and removal, and `skipped tests` to detect new skipped and un-skipped tests, as
 See this complete list of configuration options for reference:
 ```yaml
   with:
-    github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This lists all available options, and in some cases you might want to provide a different token than the default one. So this should stay for completeness.

Reply:
ah, good eye! i updated the name to `your-custom-pat` to try and emphasize that it isn't necessary to provide. happy to switch back to `${{ secrets.GITHUB_TOKEN }}` or whatever other string you want.

https://github.com/EnricoMi/publish-unit-test-result-action/pull/82/commits/05c325652cc07f8398b34deb8f50affd77eaa06a